backend/src/main/kotlin/be/osoc/team1/backend/security/ForgotPasswordUtil.kt

```diff
@@ -0,0 +1,74 @@
+package be.osoc.team1.backend.security
+
+import java.security.MessageDigest
+import java.util.SortedMap
+import java.util.UUID
+
+/**
+ * This object contains every function to manage password forgotten requests.
+ */
+object ForgotPasswordUtil {
```

Comment:
It's a bit strange in my opinion that this stuff is in a static util class when it's really just used by the `ForgotPasswordService`, can't this code basically be moved into that service class instead of putting it in a util that is only used in one place?